src/api/ssl/__tests__/Prehled-dokumentu.test.ts

```diff
@@ -0,0 +1,45 @@
+import { Ginis } from '../../../index'
+
+jest.setTimeout(20000)
```

Comment:
This a global setter and will affect other tests, please use:
````
it('Basic request', async () => {

}, 20000)
```

Why do these tests needs increased timeout? Do they send something to real service?

Reply:
This is present in all tests, I will create a separate PR to change this everywhere.

Reply:
And yes, these tests are using a real service. Ginis loves to make changes without informing us, so mocking them wouldn't help with request/response/constraint changes from their side. Therefore we have a test ginis environment, which we run these tests against, so we can verify, that they still work IRL (or as close to IRL we can get).